cfpb/consumerfinance.gov
# Buying a House: refresh dependencies and update handlebars and jquery to fix security warning

Dependabot reports a critical vulnerability for handlebars. https://github.com/cfpb/consumerfinance.gov/security/dependabot/cfgov/unprocessed/apps/owning-a-home/yarn.lock/handlebars/open

This PR updates handlebars, and jquery while we're in there. It also refreshes the yarn lock file and npm cache.

## Changes

- BAH: Update handlebars@4.7.7 from 4.7.6
- BAH: Update jquery@3.6.0 from 3.5.1
- BAH: Refresh the yarn lock file and npm cache.


## How to test this PR

1. Pull branch and run `./frontend.sh`
2. Visit http://localhost:8000/owning-a-home/explore-rates/ and make sure things are working.
